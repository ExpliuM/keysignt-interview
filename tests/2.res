syntax error, unexpected $end, expecting PATH
yylineno=1 token=IMPORT yytext=import strlen(yytext)=6
yylineno=1 token=EOF yytext= strlen(yytext)=0
