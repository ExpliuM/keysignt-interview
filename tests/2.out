sh: ./../coding-challenge: No such file or directory
